license-sh

```diff
@@ -84,21 +91,25 @@ if __name__ == "__main__":
     if ProjectType.PYTHON_PIPENV in project_types:
         runner = PythonRunner(path, silent, debug)
         dep_tree, license_map = runner.check()
+        ignored_packages = ignored_packages_map[ProjectType.PYTHON_PIPENV.value]
 
     if ProjectType.NPM in project_types:
         runner = NpmRunner(path, silent, debug)
         dep_tree, license_map = runner.check()
+        ignored_packages = ignored_packages_map[ProjectType.PYTHON_PIPENV.value]
```

Comment:
PIPENV?

Reply:
https://github.com/pypa/pipenv @S4n60w3n I don't get this comment